manifests/cf-manifest/spec/jobs_spec.rb

```diff
@@ -0,0 +1,58 @@
+RSpec.describe "the global update block" do
+
+  let(:manifest) { manifest_with_defaults }
+
+  specify "has serial false" do
+    expect(manifest["update"]["serial"]).to be false
+  end
+
+end
+
+RSpec.describe "the jobs definitions block" do
+
+  let(:jobs) { manifest_with_defaults["jobs"] }
+
+  def get_job(job_name)
+    jobs.select{ |j| j["name"] == job_name}.first
+  end
+
+  def is_serial(job_name)
+    job = get_job(job_name)
+    job["update"]["serial"]
+  end
+
+  def ordered(job1_name, job2_name)
+    i1 = jobs.index{ |j| j["name"] == job1_name }
+    i2 = jobs.index{ |j| j["name"] == job2_name }
+    i1 < i2
+  end
+
+  specify "has etcd_z1 serial" do
```

Comment:
It'd be good to group some of these related tests together and describe the reason that we care about the specific behaviour, e.g.
```ruby
describe "etcd depends on NATS" do
  specify "has nats_z1 before etcd_z1" do
    expect(ordered("nats_z1", "etcd_z1")).to be true
  end

  specify "has nats_z2 before etcd_z1" do
    expect(ordered("nats_z2", "etcd_z1")).to be true
  end
end

describe "etcd needs to start with one master node for concensus" do
  specify "has etcd_z1 serial" do
    expect(is_serial("etcd_z1")).to be true
  end

  specify "has etcd_z1 before etcd_z2" do
    expect(ordered("etcd_z1", "etcd_z2")).to be true
  end

  specify "has etcd_z1 before etcd_z3" do
    expect(ordered("etcd_z1", "etcd_z3")).to be true
  end
end
```